sw/device/silicon_creator/lib/sigverify/spx_key.c

```diff
@@ -0,0 +1,7 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#include "sw/device/silicon_creator/lib/sigverify/spx_key.h"
+
+// `extern` declarations for `inline` functions in the header.
```

Comment:
`inline`

Reply:
I'm not sure if I understand, could you please clarify?

Reply:
I probably misread. I thought there was a typo.

Reply:
SG, thanks a lot for the quick reply!